instruments/iama.yaml

```diff
@@ -1,657 +1,654 @@
 systemcard_path: .assessments[]               # append iama.yaml to 'assessments' list in system card 
-schema_version: v_1.0.0
+schema_version: 1.1.0
 
 instrument:
   name: "Impact Assessment Mensenrechten en Algoritmes (IAMA)"
   description: "Het IAMA helpt om de risico's voor mensenrechten bij het gebruik van algoritmen in kaart te brengen en maatregelen te nemen om deze aan te pakken."
-  version: 1.0
+  urn: "urn:nl:aivt:ir:1.0"
   language: "Nederlands"
```

Comment:
Make this "Dutch", or maybe better an Enum with values from ISO 639 language codes.

Reply:
Thanks, yes, totally missed this